hw/ip/entropy_src/doc/theory_of_operation.md

```diff
@@ -4,7 +4,7 @@ As already described, this IP block will collect bits of entropy for firmware or
 This revision supports only an external interface for a PTRNG noise source implementation.
 
 The first step is initialization and enabling.
-The PTRNG noise source mode is selected when the `ENABLE` field will be set.
+The PTRNG noise source mode is selected when the [`MODULE_ENABLE`](registers.html#module_enable) field is be set.
```

Comment:
Nit: Stray "is"

Reply:
Oops, thanks

The `is` was intentional, but I also meant to delete the `be`